assets/styles/index.scss

```diff
@@ -15,12 +15,17 @@ $gray-200:            #e9ecef !default;
 $border-color:        $gray-200;
 $list-inline-padding: 5px;
 $dropdown-item-padding-x: 1rem;
-{{ if not .Site.Params.fontawesome.disabled }}
-@import './fontawesome/fontawesome.scss';
-@import './fontawesome/regular.scss';
-@import './fontawesome/solid.scss';
-@import './fontawesome/brands.scss';
-{{ end }}
+{{- with .Site.Params.fontawesome -}}
```

Comment:
This breaks legacy config as we didn't check for the fontawesome config.

So it goes from working by default to it not working by default.

Reply:
Do you feel we are early enough to go for cleaner code and break for v0.15? I'm fine with either.

Reply:
Additional point. Either way I think the config should be easier.

fontawesome not configured (minimal config) -> free works
fontawesome disabled -> don't include files
fontawesome + pro -> include pro files

With this we would at the same time support the old way of doing things. So probably not breaking at all.

Reply:
I'm fine with cleaning up the code across the theme as soon as possible. Because of our plans for v0.15 I think cleaning up before releasing makes sense.